k2/csrc/tensor_ops_test.cu

```diff
@@ -305,13 +305,11 @@ static void TestIndexAdd2D() {
 
 TEST(IndexAdd, IndexAdd1D) {
   TestIndexAdd1D<float>();
-  TestIndexAdd1D<double>();
   TestIndexAdd1D<int32_t>();
 }
 
 TEST(IndexAdd, IndexAdd2D) {
   TestIndexAdd2D<float>();
-  TestIndexAdd2D<double>();
```

Comment:
@csukuangfj , I delete testing for double as I notice you have changed to use `FOR_FLOAT_AND_INT32_TYPES` in `Index1D` and `index2D`; Otherwise there was a test failure (not sure why you can run successfully as I supposed you have run them) https://github.com/k2-fsa/k2/blob/b85b9dbe2cab9a0d7d6926f1e7fb1ae2b036aeb0/k2/csrc/tensor_ops.cu#L370-L373

Reply:
Sorry, forgot to commit the changed files.

Fixed in #459